torch/utils/data/dataloader.py

```diff
@@ -30,7 +31,7 @@ def __init__(self, exc_info):
         self.exc_msg = "".join(traceback.format_exception(*exc_info))
 
 
-def _worker_loop(dataset, index_queue, data_queue, collate_fn):
+def _worker_loop(dataset, index_queue, data_queue, collate_fn, init_fn, id):
```

Comment:
Please don't use `id`, because you're shading the `id` builtin, which is quite useful. `worker_id` would be better